src/components/Resources/index.js

```diff
@@ -1,65 +1,16 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import { makeStyles } from "@material-ui/core/styles";
-import clsx from "clsx";
 import PersonalMenu from "../PersonalMenu";
-import Card from "@material-ui/core/Card";
-import CardHeader from "@material-ui/core/CardHeader";
-import CardContent from "@material-ui/core/CardContent";
-import CardActions from "@material-ui/core/CardActions";
-import Collapse from "@material-ui/core/Collapse";
-import Avatar from "@material-ui/core/Avatar";
-import IconButton from "@material-ui/core/IconButton";
-import Rating from "@material-ui/lab/Rating";
-import StarBorderIcon from "@material-ui/icons/StarBorder";
-import Typography from "@material-ui/core/Typography";
-import { red } from "@material-ui/core/colors";
-import BookmarkIcon from "@material-ui/icons/Bookmark";
-import ShareIcon from "@material-ui/icons/Share";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import MoreVertIcon from "@material-ui/icons/MoreVert";
-import Grid from "@material-ui/core/Grid";
-import TextareaAutosize from "@material-ui/core/TextareaAutosize";
-import { Button } from "@material-ui/core";
-import LinkIcon from "@material-ui/icons/Link";
 import Search from "../Search";
-
-const useStyles = makeStyles(theme => ({
-  card: {
-    maxWidth: 345
-  },
-  media: {
-    height: 0,
-    paddingTop: "56.25%" // 16:9
-  },
-  expand: {
-    transform: "rotate(0deg)",
-    marginLeft: "auto",
-    transition: theme.transitions.create("transform", {
-      duration: theme.transitions.duration.shortest
-    })
-  },
-  expandOpen: {
-    transform: "rotate(180deg)"
-  },
-  avatar: {
-    backgroundColor: red[500]
-  }
-}));
+import Grid from "@material-ui/core/Grid";
+import { ResourceCard } from "./ResourceCard";
 
 function Resources() {
   const [resources, setResources] = useState([]);
 
-  const classes = useStyles();
-  const [expanded, setExpanded] = React.useState(false);
-
-  const handleExpandClick = () => {
-    setExpanded(!expanded);
-  };
-
   useEffect(() => {
     axios
-      .get("http://localhost:3000/resources")
+      .get("http://localhost:3001/resources")
```

Comment:
Changed this endpoint because the API port is set to be `3001` in the script command (package.json). 
- https://github.com/codebuddies/react-concept/blob/master/package.json#L20

Reply:
Makes sense! Thanks.